### PR TITLE
[feat] - add reasonable warnings and more read methods

### PR DESCRIPTION
**Pull request by mur-me**
Hey guys, what I've done:
* I've added more warnings like:
  *  30s timeout on AWS API Gateway
  * info about `Content-Type: application/json` header
* added link to slack channel 
* added more read methods

Please review, and I'll point to it on our API Gateway :grin: 
Thanks for this sweet page!

### CODE REVIEW

File: components/Documentation.tsx [hunk 1]
```diff
@@ -32,6 +32,13 @@ export default function Documentation() {
           </ButtonV2Link>
         </FullWidthButtons>
 
+        <FullWidthButtons>
+          <span>If you need help with me, drop a message to my team:</span>
+          <ButtonV2Link href='https://filecoinproject.slack.com/archives/C017HM9BJ8Z'>
+            Slack Channel
```
Comment (Schwartz10) [inline components/Documentation.tsx]: Hey @mur-me I'm going to attempt to set up a bridge from our discord -> slack so we have one "place" for Glif help

File: components/Documentation.tsx [hunk 2]
```diff
@@ -92,6 +102,18 @@ export default function Documentation() {
...
+          Please, don&lsquo;t forget to add{' '}
+          <b>Content-Type: application/json</b> header to your request or
+          I&lsquo;ll give you <b>415 Unsupported Media Type error</b>
+        </WarningBox>
+
+        <WarningBox>
+          And the last side note:
+          <b> respect my 30 seconds timeout on request</b> or I&lsquo;ll throw
+          out you <b>504 Endpoint request timed out error</b>
+        </WarningBox>
```
Comment (Schwartz10) [inline components/Documentation.tsx]: Think we can leave this out

**Comment by Schwartz10** [comment]
image

File: components/Documentation.tsx (hunk 2 above)
Comment (mur-me) [reply components/Documentation.tsx]: Agreed, but if users will continue to complain on slow operations, we will add this

**Pull request merged**